src/Conversions/include/BipedalLocomotion/Conversions/CommonConversions.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file Conversions.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_COMMON_CONVERSIONS_H
+#define BIPEDAL_LOCOMOTION_COMMON_CONVERSIONS_H
+
+#include <Eigen/Dense>
+
+namespace BipedalLocomotion
+{
+namespace Conversions
+{
+
+    /**
+     * @brief Construct homogeneous transformation matrix from rotation matrix and translation vector
+     *
+     * @param rotation Eigen ref of rotation matrix
+     * @param translation Eigen ref of translation vector
+     * @param transform Eigen ref of Eigen::Matrix4d object
+     */
+    template <class Scalar>
+    void eigenRotTrans2HomogeneousTransform(const Eigen::Matrix<Scalar, 3, 3>& rotation,
+                                            const Eigen::Matrix<Scalar, 3, 1>& translation,
+                                            Eigen::Matrix<Scalar, 4, 4>& transform)
+    {
+        transform.template topLeftCorner<3,3>() = rotation;
+        transform.template topRightCorner<3,1>() = translation;
+        transform(3,3) = 1;
+        transform.template bottomLeftCorner<1,3>().setZero();
+    }
```

Comment:
I would define also 
```cpp
Eigen::Matrix<Scalar, 4, 4> eigenRotTrans2HomogeneousTransform(const Eigen::Matrix<Scalar, 3, 3>& rotation,
                                            const Eigen::Matrix<Scalar, 3, 1>& translation)
```

Reply:
Addressed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/138/commits/12d97ab9746193b054a09fd3be5e99ddd0b80836